pkg/config/execution_unit.go

```diff
@@ -88,3 +94,20 @@ func (a Application) GetExecutionUnit(id string) ExecutionUnit {
 	}
 	return cfg
 }
+
+func (cfg ExecutionUnit) GetExecutionUnitParamsAsKubernetes() KubernetesTypeParams {
+
+	infraParams := cfg.InfraParams
+	jsonString, err := json.Marshal(infraParams)
+	if err != nil {
+		zap.S().Debug(err)
+	}
+
+	params := KubernetesTypeParams{}
+	if err := json.Unmarshal(jsonString, &params); err != nil {
+		zap.S().Debug(err)
```

Comment:
I'd think this should be an `Error` log since if this happens something went wrong, right?

Reply:
yeah likely, since you only call this method with the expectation they can be marshalled correctly. Ill make that change